cmd/kube-proxy/app/server.go

```diff
@@ -153,6 +153,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 
 	fs.BoolVar(&o.CleanupAndExit, "cleanup", o.CleanupAndExit, "If true cleanup iptables and ipvs rules and exit.")
 	fs.BoolVar(&o.CleanupIPVS, "cleanup-ipvs", o.CleanupIPVS, "If true and --cleanup is specified, kube-proxy will also flush IPVS rules, in addition to normal cleanup.")
+	fs.MarkDeprecated("cleanup-ipvs", "Always flushing ipvs on --cleanup")
```

Comment:
Thanks! This text could be a bit clearer. For example `In a future release, running --cleanup will always flush IPVS rules.`

Reply:
addressed